### PR TITLE
fix handling of response from all

### PR DESCRIPTION
**Pull request by jsommers**
A couple bug fixes in getting all polls.

### CODE REVIEW

File: lib/polleverywhere/models.rb [hunk 1]
```diff
@@ -154,6 +160,16 @@ def self.get(permalink)
         from_hash(:permalink => permalink).fetch
       end
 
+      def make_current
+        http.post(:permalink => permalink).to("/my/polls/current").response do |response|
```
Comment (thoughtless) [inline lib/polleverywhere/models.rb]: Activation isn't part of our publicly supported API, so I don't think we'll be able to merge this part of the pull request at this time.

Comment (jsommers) [reply lib/polleverywhere/models.rb]: Thought I'd try :-)  Do you want a new pull req with just the fixes to Poll#all?

File: lib/polleverywhere/models.rb [hunk 2]
```diff
@@ -157,11 +157,11 @@ def self.get(permalink)
       def self.all
         PollEverywhere.http.get.from("/my/polls").as(:json).response do |response|
           ::JSON.parse(response.body).map do |hash|
-            case hash.keys.first.to_sym
+            case hash['type'].to_sym
             when MCP.root_key
               MCP.from_hash(hash)
             when FTP.root_key
-              MCP.from_hash(hash)
+              FTP.from_hash(hash)
```
Comment (thoughtless) [inline lib/polleverywhere/models.rb:192]: @jsommers I'm happy to merge these changes into the main repo. However, I think the other changes are a bit too specific to how you are using the gem for them to go into the publish rubygem.

Do you think you could change this PR to just include this one commit?

Thank you very much for your contribution.

Comment (jsommers) [reply lib/polleverywhere/models.rb:192]: That's fine.  I'll keep my other modifications separate.

Comment (thoughtless) [reply lib/polleverywhere/models.rb:192]: Either a new PR or changing this one is fine. Thanks!

Comment (jsommers) [reply lib/polleverywhere/models.rb:192]: Just created a new one, closed this one.

**Pull request closed without merging**